### PR TITLE
Feature/create classroom integration meeting

### PR DESCRIPTION
**Pull request by NicolasSCJala**

### CODE REVIEW

**Review by AJN-jalaU — APPROVED** [review]

File: src/app/modules/app/pages/classrooms/components/classroom-form/classroom-form.component.ts [hunk 1]
```diff
@@ -229,6 +241,29 @@ export class ClassroomFormComponent implements OnInit {
...
+      return {
+        id: group.get('id')?.value || 0,
+        weekdayId: group.get('weekday')?.value,
+        startTime: transform12TimeTo24Time(group.get('startingTime')?.value),
+      };
+    });
+
+    meeting.schedules = schedule;
+
+    console.log(meeting);
```
Comment (carlos-canedo-dev) [inline src/app/modules/app/pages/classrooms/components/classroom-form/classroom-form.component.ts]: Remove this console log

File: src/app/modules/app/pages/classrooms/components/classroom-form/classroom-form.component.ts [hunk 2]
```diff
@@ -199,6 +205,12 @@ export class ClassroomFormComponent implements OnInit {
     } else {
       this.classroomService.create(classroom).subscribe(this.requestHandler);
     }
+
```
Comment (carlos-canedo-dev) [inline src/app/modules/app/pages/classrooms/components/classroom-form/classroom-form.component.ts]: In the lines above you make the create or edit requests, and down here you are making the create request another time and even when the form is meant to edit the classroom, fix that please

File: src/app/shared/models/Meetings.ts
Comment (carlos-canedo-dev) [inline src/app/shared/models/Meetings.ts:1]: The file name shouldn't be plural and should be kebab case

File: src/app/modules/app/pages/classrooms/components/classroom-form/classroom-form.component.ts [hunk 3]
```diff
@@ -229,6 +241,29 @@ export class ClassroomFormComponent implements OnInit {
     return classroom;
   }
 
+  buildMeetingFromForm(data: any) {
```
Comment (carlos-canedo-dev) [inline src/app/modules/app/pages/classrooms/components/classroom-form/classroom-form.component.ts]: The process of obtaining schedules data is done in the get classroom from form method, and actually you are using this method on the submit event, so it already has this data... You can extract the schedule from the classroom in the on submit method

**Review by carlos-canedo-dev — CHANGES_REQUESTED** [review]

**Pull request closed without merging**